### PR TITLE
updated d2l-table

### PR DESCRIPTION
**Pull request by maboelsoudd2l**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -106,10 +107,9 @@
     "d2l-scroll-spy": "BrightspaceUI/scroll-spy#semver:^1",
     "d2l-sequence-viewer": "Brightspace/d2l-sequence-viewer#semver:^1",
     "d2l-simple-overlay": "Brightspace/simple-overlay#semver:^3",
-    "d2l-table": "BrightspaceUI/table#semver:^2",
+    "d2l-table": "github:BrightspaceUI/table#semver:^2",
```
Comment (dlockhart) [inline package.json]: The `github:` shouldn't be necessary

Comment (maboelsoudd2l) [reply package.json]: why did npm add it?

Comment (dlockhart) [reply package.json]: Not sure.

Comment (dlockhart) [reply package.json]: It used to be required for GitHub dependencies, but that requirement was removed a while ago.

**Pull request merged**